google-youtube.html

```diff
@@ -647,6 +747,7 @@
       this.currenttime = Math.round(this._player.getCurrentTime());
       this.currenttimeformatted = this._toHHMMSS(this.currenttime);
       this.fractionloaded = this._player.getVideoLoadedFraction();
+      this.volume = this._player.getVolume();
```

Comment:
I'd try to keep this a thin layer over the underlying YouTube Player API calls whenever possible, and this is one case where keeping it as-is makes sense to me.